cmd/virt-launcher/virt-launcher.go

```diff
@@ -421,6 +421,9 @@ func main() {
 		panic(err)
 	}
 
+	// This is needed for swtpm_setup, when running as the qemu user (swtpm_user = "qemu" in qemu.conf)
+	os.Chown("/var/lib/swtpm-localca", 107, 107)
```

Comment:
Probably would be helpfull to check and log the error here.

Also when it comes to running swtpm in session mode AFAIK the config files are searched initially in the home dir. So there is an option to avoid this `Chown`  by running the script `swtpm-create-user-config-files` (it should be shipped as part of `swtpm` RPM) as `qemu` user. This script ieventually just calls `/usr/bin/swtpm_setup --create-config-files ...`. Currently this needs to be run manually but latest libvirt does that automatically https://github.com/libvirt/libvirt/commit/c66115b6e81688649da13e00093278ce55c89cb5 . Therefore with newer libvirt probably this approach will have to be revised. I am not insisting on changing smth here now. Just FYI.

Reply:
Sounds like running that script makes more sense than chowning that directory, especially seeing the concerns related to non-root.
Hopefully we can run it as part of the virt-launcher image provisioning, so it doesn't have to happen every time we start a VMI.

Reply:
After much fighting with bazel, I was able to create a home directory for qemu...
But, even if qemu has a home, populated with `swtpm-create-user-config-files`, starting a VM still fails on trying to write to `/var/lib/swtpm-localca`... Probably need newer versions of stuff.
However, I was able to chown `/var/lib/swtpm-localca` to qemu at build time, I think it'll have to do for now, until we upgrade libvirt. Will push soon.

Reply:
Hm... Interesting. Maybe some mess with env vars... Anyway, chown at build time seems like a fine approach for now.